app/assets/stylesheets/foods.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Foods controller here.
```

Comment:
Do you need those file? Remove everything that is auto-generated and is not used